src/web/client/test/integration/errorHandler.test.ts

```diff
@@ -0,0 +1,634 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import * as vscode from "vscode";
+import * as nls from "vscode-nls";
+import Sinon, { stub, assert, spy } from "sinon";
+import { expect } from "chai";
+const localize: nls.LocalizeFunc = nls.loadMessageBundle();
+import powerPlatformExtensionContext from "../../../client/WebExtensionContext";
+import { schemaKey } from "../../schema/constants";
+import { telemetryEventNames } from "../../telemetry/constants";
+
+import {
+    queryParameters,
+    PORTALS_FOLDER_NAME_DEFAULT,
+} from "../../common/constants";
+
+import {
+    showErrorDialog,
+    removeEncodingFromParameters,
+    checkMandatoryParameters,
+    checkMandatoryPathParameters,
+    checkMandatoryQueryParameters,
+} from "../../common/errorHandler";
+describe("errorHandler", () => {
+    afterEach(() => {
+        Sinon.restore();
+    });
+    it("showErrorDialog_shouldCallShowErrorMessage", () => {
```

Comment:
I think you didnt follow the naming convention.

<img width="518" alt="image" src="https://user-images.githubusercontent.com/97606074/211750184-4a1d0772-897e-41c4-a257-ce4d8608672f.png">